src/app/community/browse/browse.component.html

```diff
@@ -1,5 +1,13 @@
 <mat-tab-group>
-  <mat-tab label="Top">Content 1</mat-tab>
+  <mat-tab label="Top">
+    <div class="browseContentWrapper">
+      <app-meme-card class="flexItem" *ngFor="let u of users"
+                     [username]=u.un
+                     [cardWidth]=u.width
```

Comment:
Why do we need to pass the width?

Reply:
uh, because idk how you wanted the width to be calculated. lol. But does your other PR fix this?

Reply:
Yes